### PR TITLE
EVG-5597: Commit Queue - Admin Nuclear Option

### PR DESCRIPTION
**Pull request by ybrill**
Button for admin to clear all items from all commit queues.

### CODE REVIEW

File: service/admin.go [hunk 1]
```diff
@@ -54,3 +54,17 @@ func (uis *UIServer) clearAllUserTokens(w http.ResponseWriter, r *http.Request)
 		gimlet.WriteJSON(w, map[string]string{})
 	}
 }
+
+func (uis *UIServer) clearAllCommitQueues(w http.ResponseWriter, r *http.Request) {
```
Comment (john-m-liu) [inline service/admin.go]: I think this should be in restV2

File: public/static/js/admin.js [hunk 2]
```diff
@@ -147,6 +147,20 @@ mciModule.controller('AdminSettingsController', ['$scope', '$window', '$http', '
       });
   }
 
+  $scope.clearCommitQueues = function(){
+    if(!confirm("This will clear the contents of all commit queues. Continue?")){
+      return
+    }
+
+    $http.delete('/admin/commit_queues').then(
```
Comment (john-m-liu) [inline public/static/js/admin.js]: If you move the route to restV2, could you add this route to rest.js and call it from there?

File: service/admin.go (hunk 1 above)
Comment (ybrill) [reply service/admin.go]: done.

File: public/static/js/admin.js (hunk 2 above)
Comment (ybrill) [reply public/static/js/admin.js]: done.

**Pull request merged**